### PR TITLE
Adding example for groups/tenants/roles

### PR DESCRIPTION
**Pull request by gabecase**

### CODE REVIEW

File: yaml-examples/faws-tenant-group-roles.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Assumptions: example assumes your identity provider includes a SAML attribute named 'groups' that includes the
+# ldap groups a user belongs too. Rename it to match your SAML attributes name.
```
Comment (TwoHeadedBoy) [inline yaml-examples/faws-tenant-group-roles.yml]: Too many 'o's in 'to'

File: yaml-examples/faws-tenant-group-roles.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      local:
+        faws:
+          groups:
+            multiValue: true
+            value:
+              - "{Ats(groups)}"
+        user:
+          domain: "{D}"
+          email: "{Pt(/saml2p:Response/saml2:Assertion/saml2:Subject/saml2:NameID)}"
+          expire: PT12H
```
Comment (TwoHeadedBoy) [inline yaml-examples/faws-tenant-group-roles.yml]: Are we supposed to put this value in quotes?

File: yaml-examples/faws-tenant-group-roles.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          roles:
+            - "{0}"
+          groups:
+            - "{1}"
+      remote:
+        -
+          multiValue: true
+          path: |
+              (
+                if (mapping:get-attributes('groups')='your_sso_ldap_group') then ('ticketing:admin/insert_tenant_id_here') else ()
```
Comment (TwoHeadedBoy) [inline yaml-examples/faws-tenant-group-roles.yml]: Should the role be generic also? Something like 'role_name/insert_tenant_id_here'?

Comment (gabecase) [reply yaml-examples/faws-tenant-group-roles.yml]: Fair point, I was thinking about keeping it for ticketing but I think your right, have a generic example for tenant roles first, then we can add some kind of ticketing specific example as its own file should we find it useful.

**Pull request merged**